contrib/intel/jenkins/run.py

```diff
@@ -157,5 +157,27 @@ def oneccltest(core, hosts, mode, util):
               .format(runoneccltest.testname))
     print("-------------------------------------------------------------------")
 
+def oneccltestgpu(core, hosts, mode, util):
+
+    runoneccltestgpu = tests.OneCCLTestsGPU(jobname=jbname,buildno=bno,
+                                         testname="oneccl GPU test", core_prov=core,
+                                         fabric=fab, hosts=hosts,
+                                         ofi_build_mode=mode, util_prov=util)
+
+    print("-------------------------------------------------------------------")
+    if (runoneccltestgpu.execute_condn):
+        print("Running oneCCL GPU examples test for {}-{}-{}" \
+              .format(core, util, fab))
+        runoneccltestgpu.execute_cmd("examples")
```

Comment:
This is nitpicky but can you change all of your "single word" strings aka strings that contain no spaces to use the single quoted 'string' instead of the double quoted "string" to match the other files?